events/bingo/README.md

```diff
@@ -1,51 +1,105 @@
 # BINGO
-Ummm...well...it's bingo!
+A Bingo game for High Fidelity. Requires setting up a domain in a very specific way. There's no easy way to deploy this Bingo game on your own domain right now.
 
-## Release Notes
+## Secrets
 
-### v1.16
+The code for this game is open source. There are two sets of secrets that you must have set up for this game to work:
+1. A `config.json` file in `<bingo root>/config/config.json` that contains the data below.
+2. A `dbInfo.json` file in `<bingo root>/webApps/dbInfo.json` that contains the data below.
```

Comment:
Small thing, but maybe move 2. To below the first example to flow into the 2nd example

Reply:
i'd like to keep it here; people are less likely to read (2) if they see it below a big block of code.